crates/driver/src/domain/competition/solution/mod.rs

```diff
@@ -497,7 +499,29 @@ pub mod error {
         InvalidClearingPrices,
         #[error(transparent)]
         Math(#[from] Math),
+        #[error("failed to calculate custom prices")]
+        CalculateCustomPrices(#[source] Trade),
+        #[error("missing native price for token {0:?}")]
+        MissingPrice(TokenAddress),
+    }
+
+    impl From<scoring::Error> for Scoring {
+        fn from(value: scoring::Error) -> Self {
+            match value {
+                scoring::Error::MissingPrice(e) => Self::MissingPrice(e),
+                scoring::Error::Math(e) => Self::Math(e),
+                scoring::Error::Scoring(e) => e,
+            }
+        }
+    }
+
+    #[derive(Debug, thiserror::Error)]
```

Comment:
Why move this here?
Looks like the error is mainly used in a different file and we only need it to know about it here to convert from it.